docs/bundles/helm.md

```diff
@@ -0,0 +1,52 @@
+# Helm Bundle Spec
+
+## Overview
+
+This document is meant to define the helm bundle format as a reference for those publishing helm bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [Bundle](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A helm bundle is a [helm chart](https://helm.sh/docs/chart_template_guide/getting_started/#charts).
+The helm chart can be created by [helm create](https://helm.sh/docs/helm/helm_create/).
+
+The currently implemented helm bundle format is the `helm+v3` format. The name of the bundle format, `helm+v3`
+combines the type of bundle (helm) with the current helm format version (v3).
+The [helm bundle provisioner](https://github.com/operator-framework/rukpak/blob/main/internal/provisioner/helm/README.md)
+is able to source `helm+v3` bundles and install them onto a Kubernetes cluster.
```

Comment:
I think conceptually, we shouldn't be talking about what's implemented or not in this doc. In my mind, this doc is simply "here the spec of a helm+v3 bundle; if you're a helm provisioner and you accept a helm+v3 bundle, you must implement unpacking/validation logic according to this spec."

Reply:
OK.  I take this out.